workflows/scRNAseq/fastq-to-matrix-10x/scrna-seq-fastq-to-matrix-10x-cellplex-tests.yml

```diff
@@ -0,0 +1,112 @@
+- doc: Test outline for scrna-seq-fastq-to-matrix-10X-cellplex.ga
+  job:
+    fastq PE collection GEX:
+      class: Collection
+      collection_type: list:paired
+      elements:
+      - class: Collection
+        type: paired
+        identifier: subsample
+        elements:
+        - identifier: forward
+          class: File
+          path: test-data/GEX_R1.fastqsanger.gz
+          filetype: fastqsanger.gz
+        - identifier: reverse
+          class: File
+          path: test-data/GEX_R2.fastqsanger.gz
+          filetype: fastqsanger.gz
+    reference genome: dm6 # To test on usegalaxy.eu dm6full
+    Barcode Size is same size of the Read: false
+    gtf:
+      class: File
+      location: https://zenodo.org/record/6457007/files/Drosophila_melanogaster.BDGP6.32.109_UCSC.gtf.gz
+      decompress: true
+      filetype: gtf
+    cellranger_barcodes_3M-february-2018.txt:
+      class: File
+      path: test-data/all_cell_barcodes_both_versions.txt
+      filetype: tabular
+    fastq PE collection CMO:
+      class: Collection
+      collection_type: list:paired
+      elements:
+      - class: Collection
+        type: paired
+        identifier: subsample
+        elements:
+        - identifier: forward
+          class: File
+          path: test-data/CMO_R1.fastqsanger.gz
+          filetype: fastqsanger.gz
+        - identifier: reverse
+          class: File
+          path: test-data/CMO_R2.fastqsanger.gz
+          filetype: fastqsanger.gz
+    sample name and CMO sequence collection:
+      class: Collection
+      collection_type: list
+      elements:
+      - class: File
+        identifier: subsample
+        location: https://zenodo.org/records/10229382/files/CMO.csv
+        filetype: csv
+    Number of expected cells (used by CITE-seq-Count)): 500
```

Comment:
I would move the ` (used by CITE-seq-Count)` in the annotation for the parameter instead of the label

Reply:
Done.